Tests/UnitTests/Purchasing/CachingProductsManagerTests.swift

```diff
@@ -0,0 +1,160 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  CachingProductsManagerTests.swift
+//
+//  Created by Nacho Soto on 9/14/22.
+
+import Nimble
+@testable import RevenueCat
+import XCTest
+
+@available(iOS 13.0, tvOS 13.0, watchOS 6.2, macOS 10.15, *)
+@MainActor
+class CachingProductsManagerTests: TestCase {
```

Comment:
To make this more explicit, should we rename this to `StoreKit1CachingProductsManagerTests`?

Reply:
This test suite is testing `CachingProductsManager` independently of what the underlying `ProductsManager` is, so it would be confusing to call it SK1 since it works just the same for SK2.